ember-mirage/src/test-support/setup-mirage.js

```diff
@@ -28,6 +28,11 @@ export function setupMirage(hooks = self, { createServer, config }) {
     }
 
     this.server = await createServer(config ?? {});
+
+    // This only works after the server was created.
+    this.server.logging =
```

Comment:
can we wrap this in an if condition checking if `mirageLogging` is in the URL?

otherwise, we override always

Reply:
Just tried again locally and now it doesn't seem to log if I do that... not sure what was going on before. Should be good now then.